docs/backends/amazon-S3.rst

```diff
@@ -17,11 +17,11 @@ in version 1.2
 Settings
 --------
 
-To use boto3 set::
+To use boto3, ensure that the Pypi packages ``boto`` and ``boto3`` are installed and set::
```

Comment:
As the original author of the boto3 version of the plugin (more like the one who ported it over from Boto), we have used this in production without boto2 for over a year.